From 6f7181c0f752418627790dca797b1fad1c56302f Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jonas.gorski@bisdn.de>
Date: Wed, 18 May 2022 16:21:02 +0200
Subject: [PATCH 06/18] onlpm: hardcode supported arches

For yocto we don't provide a dpkg, so we need to hardcode the supported
arches.

Upstream-Status: Inappropriate [Hack for building with yocto]
Signed-off-by: Jonas Gorski <jonas.gorski@bisdn.de>
---
 tools/onlpm.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tools/onlpm.py b/tools/onlpm.py
index a46a77711582..d4403c0d3b65 100755
--- a/tools/onlpm.py
+++ b/tools/onlpm.py
@@ -1001,8 +1001,7 @@ class OnlPackageManager(object):
 
 
     def __builder_arches(self):
-        arches = [ 'all', 'amd64' ]
-        arches = arches + subprocess.check_output(['dpkg', '--print-foreign-architectures'], text=True).split()
+        arches = ['amd64', 'powerpc', 'armel', 'armhf', 'arm64', 'all',]
         return arches
 
     def __build_cache(self, basedir):
-- 
2.42.0

